src/components/calcite-dropdown-group/calcite-dropdown-group.tsx

```diff
@@ -74,13 +74,17 @@ export class CalciteDropdownGroup {
 
   render() {
     const groupTitle = this.groupTitle ? (
-      <span class="dropdown-title" ref={(node) => (this.titleEl = node)}>
+      <span
+        class="dropdown-title"
+        ref={(node) => (this.titleEl = node)}
```

Comment:
Can you make this anonymous function a class method? Doing so will eliminate functions created on each render. I'll look at changing the `no-bind` rule to error to enforce this.